recipes/pycovjson/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pycovjson" %}
+{% set version = "0.3.9" %}
+{% set sha256 = "f1482516307189f8b7b8b0f3eeea645923298cea2bfb67f43f0d5e29fa2cc504" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - pycovjson-convert = pycovjson.cli.convert:main
+    - pycovjson-viewer = pycovjson.cli.viewer:main
+  noarch: python
+  skip: True  # [py2k]
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - xarray
+    - numpy
+    - pandas
+    - pymongo
+
+test:
+  imports:
+    - pycovjson
+  commands:
+    - pycovjson-convert -h
+    - pycovjson-viewer -h
+
+about:
+  about:
+    license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
It looks like there is a possible indentation issue here, and there is another `license_file` entry below still.